lis/surfacemodels/land/ac.7.2/ac72_setup.F90

```diff
@@ -0,0 +1,1359 @@
+                ! Set crop file (crop parameters are read when calling InitializeRunPart1)
+                ! Set the crop type cst in time
+                do l=1, TotalSimRuns
+                    call set_project_input(l, 'Crop_Filename', &
+                                        trim(AC72_struc(n)%ac72(t)%cropt)//'.CRO')
+                enddo
+                call CheckForKeepSWC(MultipleRunWithKeepSWC_temp, &
+                                     MultipleRunConstZrx_temp)
+                call SetSimulation_MultipleRunWithKeepSWC(MultipleRunWithKeepSWC_temp) 
+                call SetSimulation_MultipleRunConstZrx(MultipleRunConstZrx_temp)  
+                !!! Start the Program AC72
+                call SetPathNameOutp('')
+                call SetPathNameList('')
+                call SetPathNameParam('')
+                call SetPathNameProg('')
+
+
+                call GetTimeAggregationResults()
+                call GetRequestDailyResults()
+                call GetRequestParticularResults()
+
+                call SetClimRecord_DataType(0_int8)
+                call SetClimRecord_fromd(LIS_rc%sda)
+                call SetClimRecord_fromdaynr(ProjectInput(1)%Simulation_DayNr1)
+                call SetClimRecord_fromm(LIS_rc%smo)
+                call SetClimRecord_fromstring("")
+                call SetClimRecord_fromy(LIS_rc%syr)
+                call SetClimRecord_NrObs(999)
+                call SetClimRecord_tod(LIS_rc%eda)
+                call SetClimRecord_todaynr(ProjectInput(GetSimulation_NrRuns())%Simulation_DayNrN)
+                call SetClimRecord_tom(LIS_rc%emo)
+                call SetClimRecord_tostring("")
+                call SetClimRecord_toy(LIS_rc%eyr)
+                call SetClimFile('(External)')
+                call SetCO2Description('')
+                call SetProfFilefull('External')
+                call SetTemperatureRecord(GetClimRecord())
+                call SetEToRecord(GetClimRecord())
+                call SetRainRecord(GetClimRecord())
+                call SetSimulation_Storage_CropString('None')
+
+                call SetMultipleProjectDescription('undefined')
+                ProgramParametersAvailable = .false.
+                call SetFullFileNameProgramParameters("(None)")
+                call LoadProgramParametersProjectPlugIn(&
+                      GetFullFileNameProgramParameters(), &
+                             ProgramParametersAvailable)
+                call SetSimulation_MultipleRun(.true.)
+                call SetSimulation_NrRuns(TotalSimRuns) 
+                MultipleRunWithKeepSWC_temp = GetSimulation_MultipleRunWithKeepSWC() 
+                MultipleRunConstZrx_temp = GetSimulation_MultipleRunConstZrx()
+
+                ! Set AC72 soil parameters based on soiltype
+                ! Note: 2 soil layers with the same texture
+                AC72_struc(n)%ac72(t)%SoilLayer(1)%wp = WP(AC72_struc(n)%ac72(t)%soiltype) * 100
```

Comment:
Is this note still valid? Aren't *all* layers set to the same texture, not just 2? (or was 2 a hardwired max for now?)